tcms/telemetry/templates/telemetry/testing/status-matrix.html

```diff
@@ -0,0 +1,100 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load static %}
+
+{% block title %}{% trans "Testing Breakdown" %}{% endblock %}
+
+{% block contents %}
+    <div class="container-fluid container-cards-pf">
+
+        <form class="form-horizontal">
+            <div class="form-group">
+                <label for="id_select_product" class="col-md-1 col-lg-1">
+                    {% trans "Product" %}:
```

Comment:
`:` here is not needed. None of the search forms has this and you've mistakenly added it to testing-breakdown template too.